2-0-data-structures-and-algorithms/src/main/java/com/bobocode/linked_list/LinkedList.java

```diff
@@ -1,32 +1,42 @@
 package com.bobocode.linked_list;
 
+
+import java.util.NoSuchElementException;
+import java.util.Objects;
+import java.util.stream.Stream;
+
 /**
  * {@link LinkedList} is a list implementation that is based on singly linked generic nodes. A node is implemented as
- * inner static class {@link Node<T>}. In order to keep track on nodes, {@link LinkedList} keeps a reference to a head node.
+ * inner static class {@link Node<E>}.
```

Comment:
These changes should be a part of `main`. They **should not appear in this PR.**